CHANGELOG.md

```diff
@@ -5,12 +5,16 @@
 ## 🛑 Breaking changes 🛑
 
 - `memcachedreceiver`: Update metric names (#6594)
+- `spanmetricproccessor`: service.name attribute added as a default resource attribute instead of attribute in generated metrics (#6717)
```

Comment:
I don't think you need to update the changelog, the maintainers will do this upon a new release

Reply:
I saw in Zhihao's PR that there was a changelog added (see comment https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6503#pullrequestreview-829682349). @MovieStoreGuy can you confirm whether we update the changelog or the maintainers do?

Reply:
I think it is fine for us to update the changelog, we have the greatest overall idea of what has changed so I think it makes sense to add it. However, If a maintainer says otherwise, I am also fine with it.

Reply:
We would definitely prefer that you add a changelog entry, and there is CI automation that will complain if you don't!  It generally makes it easier for us to prepare releases when we don't need to track down all the changes that have been made and think about how to describe them.